emspy/query/localdata.py

```diff
@@ -99,3 +107,14 @@ def table_exists(self, table_name):
 	def file_loc(self):
 
 		return self.__dbfile
+
+	def is_db_path_correct(self, path: str):
+		'''
+		Returns true if the path represents the path for the current data file
+		'''
```

Comment:
I've been using this format: 

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

It helps when developing, I think.  Not 100% necessary (and I think docstrings are only "nice to have", since the existing emsPy code has almost none).